sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -689,6 +689,15 @@ class Column(protected[sql] val expr: Expression) extends Logging {
       println(expr.prettyString)
     }
   }
+
+  def bitwiseOR(other: Any): Column = BitwiseOr(expr, lit(other).expr)
+
+  def bitwiseAND(other: Any): Column = BitwiseAnd(expr, lit(other).expr)
+
+  def bitwiseXOR(other: Any): Column = BitwiseXor(expr, lit(other).expr)
+
+  def bitwiseNOT : Column = BitwiseNot(expr)
```

Comment:
sorry after looking at it more, I have a new round of comments.

we should move bitwiseNOT into functions.py